frontend/src/reviews/ReviewDecision.tsx

```diff
@@ -1,20 +1,20 @@
 import Done from '@mui/icons-material/Done'
 import HourglassEmpty from '@mui/icons-material/HourglassEmpty'
-import { Box, Card, Stack, Typography } from '@mui/material'
+import { Box, Card, Divider, Stack, Typography } from '@mui/material'
 import { useMemo } from 'react'
 import UserAvatar from 'src/common/UserAvatar'
-import { DecisionKeys } from 'types/interfaces'
+import { ReviewResponse } from 'types/interfaces'
 import { EntityKind } from 'types/types'
+import { formatDateString } from 'utils/dateUtils'
 
 type ReviewDecisionProps = {
-  user: string
-  decision: DecisionKeys
+  response: ReviewResponse
 }
 
-export default function ReviewDecision({ user, decision }: ReviewDecisionProps) {
-  const isApproved = useMemo(() => decision === 'approve', [decision])
+export default function ReviewDecision({ response }: ReviewDecisionProps) {
+  const isApproved = useMemo(() => response.decision === 'approve', [response.decision])
 
-  const username = user.split(':')[0]
+  const username = response.user.split(':')[0]
```

Comment:
Based off of a comment on your other PR, it think this might need to be `response.user.split(':')[1]`, but I'd double check before trusting me on that

Reply:
This is currently correct for this case as it uses the full user:user name, I'm personally ok leaving this until the entity naming stuff is done on the backed as I don't to change much more of the review code.